core/request.py

```diff
@@ -318,15 +318,28 @@ def processRequest(self) -> None:
                         res["traceback"] = traceback.format_exc()
 
                     res = json.dumps(res)
-                else:
-                    with (conf["viur.instance.core_base_path"].joinpath(conf["viur.errorTemplate"]).open() as tpl_file):
-                        tpl = Template(tpl_file.read())
-                        res = tpl.safe_substitute({
+                else:  # We render the error in html
+                    # Try to get the template from html/error/
+                    template_file = conf["viur.instance.project_base_path"].joinpath(f"html/error/{e.status}.html")
+                    if template_file.is_file():
+                        template = conf["viur.mainApp"].render.getEnv().get_template(
+                            conf["viur.mainApp"].render.getTemplateFileName(f"error/{e.status}"))
+
+                        res = template.render({
                             "error_code": e.status,
                             "error_name": translate(e.name),
                             "error_descr": e.descr,
```

Comment:
Shall we use the same dict as with #614 and #690 here? So without the "error_"-prefix?